README.adoc

```diff
@@ -15,31 +15,16 @@ images. The branches of this repository correspond to versions of the s2i-dotnet
 $ oc new-project mydemo
 
 # Add the .NET Core application
-$ oc new-app dotnet:6.0-ubi8~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnet-6.0 --context-dir app
+$ oc new-app dotnet:7.0-ubi8~https://github.com/redhat-developer/s2i-dotnetcore-ex#dotnet-7.0 --context-dir app
 
 # Make the .NET Core application accessible externally and show the url
 $ oc expose service s2i-dotnetcore-ex
 $ oc get route s2i-dotnetcore-ex
 ----
 
-== Deploy using the OpenShift Do ('odo')
```

Comment:
I've removed the `odo` instructions because the `odo` team changed their example application to their own repo:  https://github.com/devfile-samples/devfile-sample-dotnet60-basic.git.